packages/notifi-react/lib/context/NotifiTopicContext.tsx

```diff
@@ -120,16 +120,14 @@ export const NotifiTopicContextProvider: FC<PropsWithChildren> = ({
         const fusionEventId = topic.fusionEventDescriptor.id;
         const fusionEventMetadata = getFusionEventMetadata(topic);
         if (!fusionEventMetadata || !fusionEventId) return;
-        let subscriptionValueObject: InputObject[] | null = [
-          { label: '', value: '' },
-        ];
         if (!subscriptionValue) {
           const subscriptionValueOrRef =
             getFusionEventMetadata(topic)?.uiConfigOverride
               ?.subscriptionValueOrRef;
-          subscriptionValueObject = subscriptionValueOrRef
+          const subscriptionValueObject = subscriptionValueOrRef
             ? [...resolveObjectArrayRef('', subscriptionValueOrRef, inputs)]
             : null;
+          subscriptionValue = subscriptionValueObject?.[0]?.value;
```

Comment:
Sorry I know it is tedious, but would you mind to change it to plural as it is an array.
`subscriptionValueObjects`